spec/models/canonical_item_spec.rb

```diff
@@ -49,4 +49,16 @@
 
   describe "Methods >" do
   end
+
+  describe "Filtering >" do
+    describe "->by_partner_key" do
+      before(:each) do
+        CanonicalItem.delete_all
```

Comment:
Could you approach this spec slightly differently?

The `CanonicalItem`s are seeded at the beginning of the specs and `DatabaseCleaner` has been instructed not to re-create them.

Two possible paths for this would be either:
 1. Don't do `delete_all`, just grab the partner key from `CanonicalItem.first` or
 1. Set up an `after { ... }` block that calls `seed_canonical_items_for_tests`

The latter will be more DB intensive (roughly 30-40 CanonicalItems get re-seeded)

Reply:
Okay, this test has been fixed and made a tad more robust.